src/components/tables/Table.tsx

```diff
@@ -238,11 +241,18 @@ export class Table extends React.Component<ITableProps> {
         const numberOfSelectedIds: number = tableData.selectedIds ? tableData.selectedIds.length : 0;
 
         const tableBodyNode: React.ReactNode = tableData.displayedIds.map((id: string, yPosition: number): JSX.Element => {
-            const currentRowData: IData = tableData.byId[id];
+            const row: IData = tableData.byId[id];
+            const firstColumnName: string = _.keys(row)[1];

```

Comment:
not sure to understand what you want to do here, you should maybe add a comment explaining line 245